Hydration_freenrg/README.md

```diff
@@ -0,0 +1,9 @@
+# Hydration Free Energy
+A set of scripts used to reproduce the hydration free energy results of [Leoffler _et al_](10.1021/acs.jctc.8b00544) using BioSimSpace. Currently designed to be run specifically on the Yoko cluster.
+
+## Usage
```

Comment:
Perhaps we want to say how the environment was created. Presumably it's the basic BioSImSpace conda environment, but you probably added `openbabel` as an additional dependency.